catalyst-gateway/bin/src/event_db/legacy/queries/event/config.rs

```diff
@@ -0,0 +1,81 @@
+//! Config Queries
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use tracing::error;
+
+use crate::event_db::{Error, EventDB};
+
+#[async_trait]
+#[allow(clippy::module_name_repetitions)]
+/// Config Queries Trait
+pub(crate) trait ConfigQueries: Sync + Send + 'static {
+    async fn get_config(&self) -> Result<(Vec<NetworkMeta>, FollowerMeta), Error>;
+}
+use crate::event_db::Error::JsonParseIssue;
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, PartialOrd, Clone)]
+/// Network config metadata
+pub(crate) struct NetworkMeta {
+    /// Mainnet, preview, preprod
+    pub network: String,
```

Comment:
Maybe use `Network` from `cardano_chain_follower`?

Reply:
I didn't type this because the context relates to parsing raw JSON from the database and the network type from cardano is missing stuff such as partialEq. It's easier to leave it as a string.